pkg/accounting/live/live.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // Config contains configurable values for the live accounting service.
 type Config struct {
-	StorageBackend string `help:"what to use for storing real-time accounting data"`
+	StorageBackend string `help:"what to use for storing real-time accounting data" default:"plainmemory"`
```

Comment:
Any reason it's not called `inmemory` instead of `plainmemory`. Is there something that makes it more plainer than inner? :D

Reply:
@egonelbre Thats a question for @thepaul

Reply:
One potential future alternative would be something like memcache, which would still be in-memory, but would also be distributed and structured more intelligently.